spec/support/models.rb

```diff
@@ -3,6 +3,7 @@ def load_models
     define_constant :PublisherModel do
       include Mongoid::Document
       include Promiscuous::Publisher
+      include Mongoid::Attributes::Dynamic
```

Comment:
In mongoid 4, dynamic attributes are disabled by default and can be turned on per-collection with this method.